cluster/src/main/java/org/apache/iotdb/cluster/query/ClusterPlanner.java

```diff
@@ -60,4 +62,18 @@ public PhysicalPlan rawDataQueryReqToPhysicalPlan(
     return new ClusterPhysicalGenerator()
         .transformToPhysicalPlan(operator, rawDataQueryReq.fetchSize);
   }
+
+  @Override
+  public PhysicalPlan lastDataQueryReqToPhysicalPlan(
```

Comment:
Why do you need to override the method?

Reply:
ClusterPlanner extends Planner,i  add lastDataQueryReqToPhysicalPlan in Planner